docs/upgrade-testing.md

```diff
@@ -0,0 +1,140 @@
+## Upgrade testing 
+Follow below step for manual upgrade testing 
+
+1. Set environment variables to overwrite values in Makefile. You should overwrite the IMAGE_OWNER and VERSION etc values for pushing container images into your quay.io account.
+
+```
+IMAGE_OWNER ?= IMAGE_OWNER
+VERSION ?= VERSION
+```
+
+2. Add `replaces` proprty in [opendatahub-operator.clusterserviceversion.yaml](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml) and add version which you would like to upgrade with next version
+
```

Comment:
```suggestion
2. Add `replaces` property in [opendatahub-operator.clusterserviceversion.yaml](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml) and add version which you would like to upgrade with next version

```

It also would make sense to link to latest commit instead of `HEAD` of incubation branch as this might change.